react/features/unsupported-browser/components/browserLinks.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Represents hash map that contains browser download links.
+ *
+ * @type {Object}
+ */
+const BROWSER_LINKS = {
+    CHROME: 'http://google.com/chrome',
+    CHROMIUM: 'http://www.chromium.org/',
+    FIREFOX: 'http://www.getfirefox.com/',
+    SAFARI: 'https://support.apple.com/downloads/safari',
+    IE: 'https://www.microsoft.com/en-us/download/internet-explorer.aspx'
```

Comment:
It's not a long list, but still it would be good to have them in alphabetical order.